CONTRIBUTING.md

```diff
@@ -1,7 +1,71 @@
+# Rationale
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposal), or simply [editorial
+improvements](#editorial-improvements).
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals
+
+We are open to any proposal, including the introduction of new sets of
+intrinsics. We _do not_ require copyright assignment for the
+introduction of new specifications. The original contributor will
+retain the copyright.
```

Comment:
It seems to me that the copyright assignment question could also be raised for "bug fixes". Would it be better to have this in a separate, orthogonal, section? Maybe the "Contribution agreement" section?